### ISSUE TITLE
[BITV] The apps view represents a list of elements and should be implemented as such

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#40015]
This issue is a same as https://github.com/nextcloud/photos/issues/1665 
See `settings/apps/files` or any other view which is not a list view.

The list of the apps should be a right semantic list.

![Screenshot from 2023-08-23 11-54-54](https://github.com/nextcloud/server/assets/6078378/bfd414d5-6ff1-4ae7-a5c7-0547a2b19434)

**Comment by JuliaKirschenheuter** [comment]
@michaelnissenbaum am i right about this issue? Should the apps be presented as a right semantic list?

**Comment by michaelnissenbaum** [comment]
HI @JuliaKirschenheuter. You're right, it's best to use a semantic list here. Another option would be to use headings, or both simultaneously - a list and headings.
